dotcom-rendering/src/layouts/FrontLayout.tsx

```diff
@@ -683,9 +689,34 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 					pillars={NAV.pillars}
 					urls={front.nav.readerRevenueLinks.header}
 					editionId={front.editionId}
-					contributionsServiceUrl="https://contributions.guardianapis.com" // TODO: Pass this in
+					contributionsServiceUrl={contributionsServiceUrl}
 				/>
 			</Section>
+
+			<BannerWrapper data-print-layout="hide">
+				<Island deferUntil="idle" clientOnly={true}>
+					<StickyBottomBanner
+						contentType={front.config.contentType}
+						contributionsServiceUrl={contributionsServiceUrl}
+						idApiUrl={front.config.idApiUrl}
+						isMinuteArticle={false}
+						isPaidContent={!!front.config.isPaidContent}
+						isPreview={front.config.isPreview}
+						isSensitive={front.config.isSensitive}
+						keywordIds={front.config.keywordIds} // a front doesn't really have tags, but frontend generates a keywordId
+						pageId={front.pressedPage.id}
+						section={front.config.section}
```

Comment:
Weirdly it looks like on articles both `section` and `sectionName` are passed as props. I'm not sure what the difference is - is one a human readable string and the other an ID of some kind maybe? From `StandardLayout`:

```
								section={article.config.section}
								sectionName={article.sectionName}
```

Braze uses `sectionName` for its limited section targeting. I wonder if there's an equivalent we could pass through for fronts? On frontend it's fetched like this: `const section = config.get('page.section');`

Reply:
I can't see `sectionName` being used anywhere on frontend.
It doesn't appear to be available currently in DCR for fronts, I wonder if we can migrate to using `section`, which is the section ID

Reply:
This is being addressed in a separate PR: https://github.com/guardian/dotcom-rendering/pull/8222

Reply:
I've rebased, and now we only pass in a `sectionId` prop